lua/starfall/libs_sv/prop.lua

```diff
@@ -413,6 +413,7 @@ function props_library.createSent(pos, ang, class, frozen, data)
 	checkpermission(instance,  nil, "prop.create")
 
 	checkluatype(class, TYPE_STRING)
+	checkluatype(frozen, TYPE_BOOL)
 	frozen = frozen and true or false
```

Comment:
```suggestion
	if frozen~=nil then checkluatype(frozen, TYPE_BOOL) end
```

Reply:
Also change `-- @param boolean frozen True to spawn frozen` to `-- @param boolean? frozen True to spawn frozen (default false)`